jobs.groovy.j2

```diff
@@ -90,6 +90,28 @@
 {# JINJA2 MACROS DEFINED BELOW                                               #}
 {# --------------------------------------------------------------------------#}
 
+{% macro updateGitHubStatus(status, description, branch, dashProject,
+                            dashBranchName)                                 -%}
+"""
+#!/bin/bash
+set +x
+{# Read the GitHub credentials                                               #}
+. /etc/slave_config/github_status.sh
+
+{# Send a request to Github to update the commit with the build status       #}
+REMOTE_GIT_COMMIT=`git rev-list --max-count=1 upstream/{{branch}}`
+BUILD_URL="\$JENKINS_URL/job/{{dashProject}}/job/{{dashBranchName}}/"
+PAYLOAD="{\\"state\\": \\"{{ status }}\\", \\"target_url\\": \\"\$BUILD_URL\\", \\"description\\": \\"{{ description }}\\", \\"context\\": \\"jenkins-multijob\\" }"
+curl \\
+  -H "Content-Type: application/json" \\
+  -H "Authorization: token \$GITHUB_STATUS_API_TOKEN" \\
+  -X POST \\
+  --data "\$PAYLOAD" \\
+  https://api.github.com/repos/ClusterHQ/flocker/statuses/\$REMOTE_GIT_COMMIT
```

Comment:
I think you might be able to use some jinja here {{ cfg.project }}
from: https://github.com/ClusterHQ/flocker/blob/master/build.yaml#L56

something along the lines of:
https://api.github.com/repos/{{ cfg.project }}/statuses/\$REMOTE_GIT_COMMIT